### PR TITLE
@zephraph => [Palette] Swap /Styleguide/Elements for @artsy/palette versions

### PR DESCRIPTION
**Pull request by damassi**
Depends on https://github.com/artsy/palette/pull/71 and https://github.com/artsy/reaction/pull/1227

Moves out our Elements from Reaction into Palette as well as updates call-sites. 

Follow-up PR will remove the elements directory completely, but only after Volt is updated to point to latest Palette version

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit e3bc6c6f559b29c4280330da23b3ab38b550eca4

https://deploy-preview-1216--artsy-reaction.netlify.com

File: package.json [hunk 1]
```diff
@@ -70,9 +70,10 @@
     "@babel/preset-stage-3": "7.0.0-beta.54",
     "@babel/preset-typescript": "7.0.0-beta.54",
     "@playlyfe/gql": "^2.3.2",
-    "@storybook/addon-info": "^3.4.6",
-    "@storybook/addon-options": "^3.2.1",
-    "@storybook/react": "^3.2.12",
+    "@storybook/addon-info": "^3.4.10",
+    "@storybook/addon-options": "^3.4.10",
+    "@storybook/addons": "^3.4.10",
```
Comment (damassi) [inline package.json]: fixes peer dep issue

**Comment by zephraph** [comment]
All the conflicts!

image

**Comment by damassi** [comment]
grrrr

**Comment by zephraph** [comment]
Whew! This is huge. Looks great though! It's a big step, amazing work. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**